### ISSUE TITLE
Issue with text file creation.

### ISSUE THREAD

**Issue by KShaver** [omgmog/photoshop-text-export#1]
I tried this code and it doesn't appear to be creating the .txt file from the text layer.
There are no issues when I run the script in Photoshop CS5.
The save window appears.. but no file is created.

Any suggestions?

**Comment by jm3** [comment]
@Blumed any luck with this?

**Comment by Blumed** [comment]
@jm3 Yes and no. There is a document that shows you all the hooks and how to use them. I never saved it so I am not sure where it is now, but it exists. I ended up stripping out the parts that caused errors and converting everything into html and using css to style it. I would then use the browser's default find function to navigate up and down the document.

**Comment by Blumed** [comment]
I will share with you what I did and this worked for me [Here](https://www.dropbox.com/s/43enrarq1b0wgfs/Text%20Export.jsx?dl=0).
* If the person who made the file has a super crazy layering naming convention. This plugin becomes useless. 
*  I had to remove the font style details, because those hooks are the culprits of the errors.
* If you purely need the text and you need to navigate through it then my change will help you out.
@jm3 Hope this helps
